apps/webstore/src/app/shared/components/products/family-tree/family-tree-design/family-tree-design.component.ts

```diff
@@ -88,8 +88,8 @@ export class FamilyTreeDesignComponent
   public context: CanvasRenderingContext2D;
 
   canvasResolution = {
-    height: 4000,
-    width: 4000,
+    height: 1000,
+    width: 1000,
   };
```

Comment:
This resolution is too low. It decreases the screenshot quality so the text becomes less visible on the print. If we have the design res be 4x4 k while the design is imutable, and 1x1 k whilte mutable, that would work. But i need to be able to get a higher resolution screenshot of the design.

Reply:
I say lets test the canvas export tools before we gimp the users. svg export from canvas.

Reply:
Automated export of the image into an SVG is a different issue. This issue and pull request is specifically about improving the performance of the current system